Dockerfile

```diff
@@ -1,11 +1,24 @@
-FROM ubuntu:22.04
+FROM ubuntu:jammy
 ARG VERSION
 
-RUN apt update && apt install -y \
+ENV TZ=UTC
+
+RUN set -eux; \
+# install python
+    apt-get update; \
+    DEBIAN_FRONTEND=noninteractive apt-get full-upgrade -y; \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

Comment:
don't we miss `;` or `&&` at the end ?

Reply:
recommentation says to use ; instead of &&

Reply:
ok, you do not have any at the end of the line

Reply:
true lol

Reply:
lol no ... it is installing packages on next lines

Reply:
I see now, we do double export of the variable, that one tricked me